### PR TITLE
spelling checked

### PR DESCRIPTION
**Pull request by Actionmaker1**
Checked spelling, changed sentences in the hopefully right syntax

### CODE REVIEW

File: app/views/pages/welcome.html.erb [hunk 1]
```diff
@@ -6,5 +6,5 @@
   <%= image_tag "AI_pic.jpg", class: "rounded mx-auto d-block", :style => "width: 75%" , alt: "Sandro AI" %>
 </div>
 <h2>Hello, I'm Sandro.</h2>
-  <p>Did you know that we spend a <i>third</i> of our life - or <strong>90.000 hours</strong> - on the job?<br>In short, it makes a lot of sense to find the <i>right</i> one. We will help you to find your true path.</p>
+  <p>Did you know that we spend a <i>third</i> of our life - or <strong>90.000 hours</strong> - on the job?<br>In short, it makes a lot of sense to find the <i>right</i> one. We will help you to find your <i><b> Truepath</i></b>. My algorhythem will help you to find the career meant for you! </p>
```
Comment (dwiehoff) [inline app/views/pages/welcome.html.erb]: algorithm 🤓

File: app/views/professions/compare.html.erb [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <div class="second">
       <a href="#" class="btn btn-outline-primary">Softare Engineer</a>
       <h4>Description</h4>
-      <p>Binding business wit hprogrammer knowhow</p>
+      <p>Binding business with programmer knowhow</p>
```
Comment (dwiehoff) [inline app/views/professions/compare.html.erb]: * `programming`?
* `know-how`

File: db/seeds.rb [hunk 3]
```diff
@@ -124,12 +124,12 @@
 # Create questions
 q1 = Question.create!(text: "What sounds more appealing to you?")
 q2 = Question.create!(text: "What sounds more appealing to you?")
-q3 = Question.create!(text: "Do you prefer to work in a creative or more in an analytical way?")
-q4 = Question.create!(text: "Do you prefer specialisation or in a certain area or to be more of a generalist?")
+q3 = Question.create!(text: "Do you prefer to work in a creative or in an more analytical way?")
```
Comment (dwiehoff) [inline db/seeds.rb]: `a more`

File: db/seeds.rb [hunk 4]
```diff
@@ -124,12 +124,12 @@
...
+q4 = Question.create!(text: "Do you prefer specialisation in a certain area or to be more of a generalist?")
 q5 = Question.create!(text: "Would you call yourself an Entrepreneurial or more of an Investigative thinker?")
 q6 = Question.create!(text: "What sounds more appealing to you?")
-q7 = Question.create!(text: "Are you thinking/planing things trough or dealing with problems when they come up?")
-q8 = Question.create!(text: "Do you like working one main task/ product after another or do you want to experience?")
+q7 = Question.create!(text: "Are you thinking/planing things through or rather deal with problems when they come up?")
```
Comment (dwiehoff) [inline db/seeds.rb]: * planning
* planning things through (what does that mean?)

File: db/seeds.rb [hunk 5]
```diff
@@ -124,12 +124,12 @@
...
+q8 = Question.create!(text: "Do you like working on one main task/ product after another or do you want to experience diversity in your worklife?")
```
Comment (dwiehoff) [inline db/seeds.rb]: `task/product`

Comment (dwiehoff) [reply db/seeds.rb]: `at a time`

**Review by Louispau — APPROVED** [review]

**Pull request merged**